src/views/VolunteerDetailView.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div v-if="volunteer" class="flex overflow-hidden">
+    <div class="flex-none flex flex-col justify-start items-center p-8 w-[400px] overflow-auto">
+      <header class="flex flex-col items-center gap-3">
+        <!-- Avatar -->
+        <div class="border border-[3px] border-green-300 rounded-full p-2">
+          <img
+            :src="baseUrl + '/files/' + volunteer.avatar"
+            alt="Avatar von ausgewähltem Freiwilligen"
+            class="aspect-square w-[110px] object-cover rounded-full"
+          />
+        </div>
+        <!-- Name -->
+        <h2 class="pt-4 text-lg font-medium">
+          {{ volunteer.person.firstname }} {{ volunteer.person.lastname }}
+        </h2>
+        <!-- Project -->
+        <p class="text-sm">
+          Einsatzstelle: <span class="font-medium">Kindergarten Kleeblatt</span>
+        </p>
+        <!-- Status-Pill -->
+        <div class="flex flex-row border border-2 border-voloblue-100 rounded-lg text-sm">
```

Comment:
also line 6 in class: "border-2" and "border-[3px]" is enough, doesnt need another "border"